release/README.md

```diff
@@ -0,0 +1,41 @@
+# openpilot releases
+
+
+## terms
+
+`channel` - a named version of openpilot (git branch, casync caidx) which receives updates<br>
```

Comment:
shouldn't need to use a `<br>` in a markdown file. just use bullets or something